<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>h264FindText</class>
 <widget class="QMainWindow" name="h264FindText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="findLabel">
            <property name="text">
             <string>查找内容(二进制):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="findLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="directGroupBox">
          <property name="title">
           <string>方向</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QRadioButton" name="upRadioButton">
                 <property name="text">
                  <string>向上(U)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="downRadioButton">
                 <property name="text">
                  <string>向下(D)</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QCheckBox" name="continueFindNextCheckBox">
               <property name="text">
                <string>连续查找</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="findNextPushButton">
          <property name="text">
           <string>查找下一个(F)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closePushButton">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
